unity-renderer/Assets/Scripts/MainScripts/DCL/Models/AvatarAssets/WearableItem.cs

```diff
@@ -231,6 +238,14 @@ public string[] GetHidesList(string bodyShapeType)
                 : hides.Concat(SKIN_IMPLICIT_CATEGORIES).Distinct().ToArray();
         }
 
+        // Old UPPER_BODY Wearables are incompatible with the new Hands category,
+        // so we apply this rule to hide the hands by default
+        bool isOrHidesUpperBody = hides.Contains(Categories.UPPER_BODY) || data.category == Categories.UPPER_BODY;
+        bool removesDefault = data.removesDefaultHiding?.Contains(Categories.HANDS) ?? false;
```

Comment:
its hard to understand why the naming of `removesDefault`

Reply:
Yeah the concept of this `removesDefaultHiding` is super corner case-y, it's the workaround we found for hiding the base hands by default and then removing that default if the upper body is a newer version.